plugins/tiddlywiki/multiwikiserver/modules/commands/mws-add-user.js

```diff
@@ -0,0 +1,58 @@
+/*\
+title: $:/plugins/tiddlywiki/multiwikiserver/commands/mws-add-user.js
+type: application/javascript
+module-type: command
+
+Command to create users and grant permission
+
+\*/
+(function(){
+
+  /*jslint node: true, browser: true */
+  /*global $tw: false */
+  "use strict";
+  
+  var crypto = require("crypto");
+  
+  exports.info = {
+    name: "mws-add-user",
+    synchronous: false
+  };
+  
+  var Command = function(params,commander,callback) {
+    this.params = params;
+    this.commander = commander;
+    this.callback = callback;
+  };
+  
+  Command.prototype.execute = function() {
+    var self = this;
+  
+    if(this.params.length < 2) {
+      return "Usage: --mws-add-user <username> <password> [email]";
+    }
+  
+    if(!$tw.mws || !$tw.mws.store || !$tw.mws.store.sqlTiddlerDatabase) {
+      return "Error: MultiWikiServer or SQL database not initialized.";
+    }
+  
+    var username = this.params[0];
+    var password = this.params[1];
+    var email = this.params[2] || username + "@tiddlywiki.com";
```

Comment:
If `username + @tiddlywiki.com` is a placeholder, imo it should be `example.com` -- Otherwise it means, that tiddlywiki.com also provided an email-service. Which is not the case

Reply:
Perhaps use `example.com` for the moment?